starter-code/css/main.css

```diff
@@ -1,8 +1,141 @@
 /*
 Colors:
-
 Text: 1A1A1A
 Green: #00B172
 White: #FFF
-
 */
+html{
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+body{
+  text: #1A1A1A;
+  padding-top: 80px;
+}
+
+.top-bar{
+  position: fixed;
+  height: 110px;
+  width: 100%;
+  top: 0px;
+  background-color: white;
+}
+
+.green-logo{
+  width: 250px;
+  float: left;
+  margin-left: 50px;
+  margin-top: 20px;
+}
+
+nav ul{
+  float: right;
+  text-align: center;
+}
+
+nav li{
+  font-size: 24px;
+  line-height: 130px;
+  padding-left: 30px;
+  display: inline;
+}
+
+.nav-list {
+  list-style: none;
+  width: 500px;
+  margin-right: 75px;
+  margin-top: 0px;
+}
+
+.hero-section{
+  background-color: green;
+  height: 700px;
+  background-size: cover;
+  background: no-repeat;
+}
+
+.landing-image{
+  height: 100%;
+  width: 100%;
+}
+
+.main-heading{
+  color: white;
+  margin-left: 0px;
+  margin-top: -500px;
+  text-align: center;
+  font-size: 72px;
+}
+
+.sub-heading{
+  color: white;
+  text-align: center;
+  line-height: 10px;
+  font-weight: 200;
+  font-size: 30px;
+}
+
+.center-section{
+  background-color: dodgerblue;
+}
+
+.center-section-group{
+
+}
+
+.millions-of-songs{
+
+}
+
+.music-icon{
+  height: 100px;
+  width: 100px;
```

Comment:
if you have multiple images or tags in general that use the same styling, you can just use the same class name for all the tags you want to target and it helps to keep your css clean so as to not repeat code over multiple times.